purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1307,11 +1307,11 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             if (purchase.purchaseState == RevenueCatPurchaseState.PURCHASED) {
                 billing.querySkuDetailsAsync(
                     productType = purchase.type,
-                    skus = setOf(purchase.sku),
-                    onReceive = { skuDetailsList ->
+                    skus = purchase.skus.toSet(),
+                    onReceive = { productDetailsList ->
                         postToBackend(
                             purchase = purchase,
-                            productDetails = skuDetailsList.firstOrNull { it.sku == purchase.sku },
+                            productDetails = productDetailsList.takeUnless { it.isEmpty() }?.get(0),
```

Comment:
another place where i'm kinda confused on the behavior here... is this defined somewhere by google? that we pass all the skus but only the first productdetails?

Reply:
This is just a patch to mimic the current behavior where there is always just one product ID. This makes sure it works, although incorrectly because we would only post the details of one of the products being purchased if someone purchases more than one at the same time.